### ISSUE TITLE
parametrize USBMUXD_SOCKET_ADDRESS

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/mcloud-device#138]
```
docker logs -f device-iPhone_8_Plus-d6afc6b3a65584ca0813eb8957c6479b9b6ebb11
2023/08/04 11:49:12 socat[19] E connect(5, AF=2 172.18.0.7:22, 16): Connection refused
{"err":"Failed getting devicelist: read unix @-\u003e/var/run/usbmuxd: read: connection reset by peer","level":"fatal","msg":"failed getting device list","time":"2023-08-04T11:49:12Z"}
2023/08/04 11:49:12 socat[28] E connect(5, AF=2 172.18.0.7:22, 16): Connection refused
{"err":"Failed getting devicelist: EOF","level":"fatal","msg":"failed getting device list","time":"2023-08-04T11:49:12Z"}
Device is not available!
Exiting without restarting...
(base) build@mcloud:~/tools/vd/appium$
(base) build@mcloud:~/tools/vd/appium$
(base) build@mcloud:~/tools/vd/appium$
(base) build@mcloud:~/tools/vd/appium$
(base) build@mcloud:~/tools/vd/appium$ docker ps -a | grep d6afc6b3a65584ca0813eb8957c6479b9b6ebb11
47b39a0e0694   public.ecr.aws/zebrunner/mcloud-device:2.4.7   "/bin/sh -c 'bash /o…"   2 minutes ago    Exited (0) 2 minutes ago                                                                                                                                                  device-iPhone_8_Plus-d6afc6b3a65584ca0813eb8957c6479b9b6ebb11
adbdc724ad5b   public.ecr.aws/zebrunner/uploader:3.1          "/entrypoint.sh"         2 minutes ago    Up 2 minutes                                                                                                                                                              device-iPhone_8_Plus-d6afc6b3a65584ca0813eb8957c6479b9b6ebb11-uploader
7b744507ac15   public.ecr.aws/zebrunner/appium:2.0-goios      "/opt/entrypoint/ent…"   2 minutes ago    Up About a minute (healthy)   4723/tcp, 0.0.0.0:7431->7431/tcp, :::7431->7431/tcp                                                                                         device-iPhone_8_Plus-d6afc6b3a65584ca0813eb8957c6479b9b6ebb11-appium
```

in above use-case due to the several appium restarts on pairing STF wasn't able to detect appium and exited without restart. As only pairing fixed - stf still exited. 
Let's investigate later and if it is only for pairing step then keep as is but if it is repro inside production usage then definitely - revert!

**Comment by vdelendik** [comment]
on linux we have share usbmuxd via `appium:22` and socat tcp listener proxy. For the MacOS we have to bind onto the server usbmuxd/socat address. Let's parametrize this value to be able to manage based on usage OS keeping by default `appium:22`

**Comment by vdelendik** [comment]
fix delivered in mcloud-device:2.5.1 which is reused in mcloud-agent with MacOS support

**Comment by dhreben** [comment]
Verified.

`34136832    548   ??  S    12:29PM   0:01.35 /usr/local/bin/socat TCP-LISTEN:2222,reuseaddr,fork UNIX-CONNECT:/var/run/usbmuxd`
